web/packages/teleport/src/Discover/Shared/LabelsCreater/LabelsCreater.tsx

```diff
@@ -174,30 +175,15 @@ export function LabelsCreater({
           );
         })}
       </Box>
-      <ButtonText
-        onClick={addLabel}
+      <ButtonTextWithAddIcon
         css={`
-          padding-left: 0px;
-          &:disabled {
-            .icon-add {
-              opacity: 0.35;
-            }
-            pointer-events: none;
-          }
+          margin-top: -2px;
```

Comment:
i don't think this is respected, i'd remove it. it was to push up the plus icon so it's more aligned, but it's so slight you can't really tell

Reply:
removed in https://github.com/gravitational/teleport/pull/36103/commits/afc2f96d25910bae3538d5102ec04725baab8bcf